CONTRIBUTING.md

```diff
@@ -1,143 +1,177 @@
-<h1><span lang="en">Contributing</span> / <span lang="de">Mitarbeit</span></h1>
+# Contributing :tada:
+Thank you for wanting to help out! :heart:
 
-<span lang="en">Thank you for wanting to help out!</span> <em lang="de">Danke dass du mithelfen möchtest!</em>
+Danke dass du mithelfen möchtest!
+Die deutsche Version dieses Dokuments findest du [hier](./CONTRIBUTING_DE.md).
 
-<em lang="de">Die <a href="#deutsch">deutsche Übersetzung</a> findest du weiter unten.</em>
+# English :milky_way:
 
-## English
+## [Code of conduct](https://www.ecamp3.ch/en/code-of-conduct) :page_with_curl:
 
-### [Code of conduct](https://www.ecamp3.ch/en/code-of-conduct)
+## Workflow :gear:
+This is a basic overview of the workflow, i.e. how we work with the code of eCamp v3. More information about how to set up a development environment on your computer is in the [wiki](https://github.com/ecamp/ecamp3/wiki/installation).
+If something about the setup is unclear, or you run into an error, there are [discussions](https://github.com/ecamp/ecamp3/discussions) on GitHub for you to ask questions and ask for help. :computer:
```

Comment:
Replace this sentence with a Discord invite link: https://discord.gg/tdwtRytV6P
We are sunsetting discussions for now and handling setup support on Discord.